storage/innobase/os/os0file.cc

```diff
@@ -10743,25 +10755,25 @@ Encryption::decrypt_log(
 #endif
 
 		/* Decrypt block */
-		ret = decrypt_log_block(type, ptr, dst);
+		ret = decrypt_log_block(type, ptr, dst, offset);
```

Comment:
add doxygen comment for decrypt_log_block too (offset param)

Reply:
Another interesting thing I found is that offset is not used at all by decrypt_log_block? :(

What is the significance of this parameter? It seems it caused rippling effect (to callers) and only to be unused?